### PR TITLE
New Stellar RPC: SetWalletAccountAsDefaultLocal

### PR DESCRIPTION
**Pull request by zapu**
This PR adds an RPC that changes primary account in a bundle and posts the bundle with user chain link.

### CODE REVIEW

**Comment by zapu** [comment]
What would be an easy way to see if sigchain links are posted?

File: go/stellar/stellar.go [hunk 1]
```diff
@@ -792,3 +792,33 @@ func ChangeAccountName(m libkb.MetaContext, accountID stellar1.AccountID, newNam
 	}
 	return remote.Post(m.Ctx(), m.G(), nextBundle)
 }
+
+func SetAccountAsPrimary(m libkb.MetaContext, accountID stellar1.AccountID) (err error) {
```
Comment (mlsteele) [inline go/stellar/stellar.go]: might as well assert `!accountID.IsNil()`

**Comment by mlsteele** [comment]
@zapu if you mean checking if sigchain links are posted for tests. This could be good enough. Loading a user's address this way gets it from their sigchain.
```
	t.Logf("Lookup the public stellar address as another user")
	u0, err := tcs[1].G.LoadUserByUID(tcs[0].G.ActiveDevice.UID())
	require.NoError(t, err)
	addr := u0.StellarWalletAddress()
```

**Comment by zapu** [comment]
Done! thanks a lot

**Pull request merged**